R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1502,6 +1502,12 @@ test_that("column functions", {
   result <- collect(select(df, sort_array(df[[1]])))[[1]]
   expect_equal(result, list(list(1L, 2L, 3L), list(4L, 5L, 6L)))
 
+  # Test flattern
```

Comment:
not a big deal at all but I'd say `flattern()` for consistency. I think detail and consistency are the key ..